Prerequisite steps not working for latest main

**Describe the bug**
When running the `make install-tools` command, I get the follow error : 
```shell
#0 35.12 wget https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v5.0.0/kustomize_v5.0.0_linux_amd64.tar.gz | tar xz -C /tmp
#0 35.12 --2023-03-13 18:14:33--  https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v5.0.0/kustomize_v5.0.0_linux_amd64.tar.gz
#0 35.12 Resolving github.com (github.com)... 140.82.113.4
#0 35.13 Connecting to github.com (github.com)|140.82.113.4|:443... connected.
#0 35.23 HTTP request sent, awaiting response... 302 Found
#0 35.42 Location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/133067498/4f5c4bbf-9f55-47e2-9c90-fd411722c4a1?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20230313%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230313T181433Z&X-Amz-Expires=300&X-Amz-Signature=1462f1a30d2f70c3d522c77ba2ec6f6052e36ef568d09d91560a201c52d2a9c6&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=133067498&response-content-disposition=attachment%3B%20filename%3Dkustomize_v5.0.0_linux_amd64.tar.gz&response-content-type=application%2Foctet-stream [following]
#0 35.42 --2023-03-13 18:14:33--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/133067498/4f5c4bbf-9f55-47e2-9c90-fd411722c4a1?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20230313%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230313T181433Z&X-Amz-Expires=300&X-Amz-Signature=1462f1a30d2f70c3d522c77ba2ec6f6052e36ef568d09d91560a201c52d2a9c6&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=133067498&response-content-disposition=attachment%3B%20filename%3Dkustomize_v5.0.0_linux_amd64.tar.gz&response-content-type=application%2Foctet-stream
#0 35.42 Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.109.133, 185.199.110.133, 185.199.111.133, ...
#0 35.42 Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.109.133|:443... connected.
#0 35.46 HTTP request sent, awaiting response... 200 OK
#0 35.52 Length: 5880512 (5.6M) [application/octet-stream]
#0 35.52 Saving to: 'kustomize_v5.0.0_linux_amd64.tar.gz'
#0 35.52 
#0 35.52      0K .......... .......... .......... .......... ..........  0% 3.98M 1s
#0 35.53     50K .......... .......... .......... .......... ..........  1% 8.12M 1s
#0 35.53    100K .......... .......... .......... .......... ..........  2% 13.7M 1s
#0 35.54    150K .......... .......... .......... .......... ..........  3% 4.81M 1s
#0 35.55    200K .......... .......... .......... .......... ..........  4% 24.9M 1s
#0 35.55    250K .......... .......... .......... .......... ..........  5% 28.8M 1s
#0 35.55    300K .......... .......... .......... .......... ..........  6% 28.7M 1s
#0 35.55    350K .......... .......... .......... .......... ..........  6% 8.67M 1s
#0 35.56    400K .......... .......... .......... .......... ..........  7% 21.2M 1s
#0 35.56    450K .......... .......... .......... .......... ..........  8% 14.0M 1s
#0 35.56    500K .......... .......... .......... .......... ..........  9% 32.0M 0s
#0 35.57    550K .......... .......... .......... .......... .......... 10% 23.2M 0s
#0 35.57    600K .......... .......... .......... .......... .......... 11% 31.7M 0s
#0 35.57    650K .......... .......... .......... .......... .......... 12% 28.5M 0s
#0 35.57    700K .......... .......... .......... .......... .......... 13% 28.4M 0s
#0 35.57    750K .......... .......... .......... .......... .......... 13% 28.0M 0s
#0 35.58    800K .......... .......... .......... .......... .......... 14% 21.5M 0s
#0 35.58    850K .......... .......... .......... .......... .......... 15% 25.4M 0s
#0 35.58    900K .......... .......... .......... .......... .......... 16% 28.6M 0s
#0 35.58    950K .......... .......... .......... .......... .......... 17% 28.4M 0s
#0 35.58   1000K .......... .......... .......... .......... .......... 18% 27.8M 0s
#0 35.58   1050K .......... .......... .......... .......... .......... 19% 28.4M 0s
#0 35.59   1100K .......... .......... .......... .......... .......... 20% 28.6M 0s
#0 35.59   1150K .......... .......... .......... .......... .......... 20% 25.6M 0s
#0 35.59   1200K .......... .......... .......... .......... .......... 21% 21.6M 0s
#0 35.59   1250K .......... .......... .......... .......... .......... 22% 31.4M 0s
#0 35.59   1300K .......... .......... .......... .......... .......... 23% 25.9M 0s
#0 35.60   1350K .......... .......... .......... .......... .......... 24% 28.4M 0s
#0 35.60   1400K .......... .......... .......... .......... .......... 25% 27.9M 0s
#0 35.60   1450K .......... .......... .......... .......... .......... 26% 28.5M 0s
#0 35.60   1500K .......... .......... .......... .......... .......... 26% 25.0M 0s
#0 35.60   1550K .......... .......... .......... .......... .......... 27% 29.1M 0s
#0 35.60   1600K .......... .......... .......... .......... .......... 28% 21.4M 0s
#0 35.61   1650K .......... .......... .......... .......... .......... 29% 25.7M 0s
#0 35.61   1700K .......... .......... .......... .......... .......... 30% 28.4M 0s
#0 35.61   1750K .......... .......... .......... .......... .......... 31% 28.5M 0s
#0 35.61   1800K .......... .......... .......... .......... .......... 32% 31.4M 0s
#0 35.61   1850K .......... .......... .......... .......... .......... 33% 25.6M 0s
#0 35.62   1900K .......... .......... .......... .......... .......... 33% 28.6M 0s
#0 35.62   1950K .......... .......... .......... .......... .......... 34% 28.2M 0s
#0 35.62   2000K .......... .......... .......... .......... .......... 35% 21.5M 0s
#0 35.62   2050K .......... .......... .......... .......... .......... 36% 25.4M 0s
#0 35.62   2100K .......... .......... .......... .......... .......... 37% 28.5M 0s
#0 35.62   2150K .......... .......... .......... .......... .......... 38% 27.0M 0s
#0 35.63   2200K .......... .......... .......... .......... .......... 39% 28.2M 0s
#0 35.63   2250K .......... .......... .......... .......... .......... 40% 28.3M 0s
#0 35.63   2300K .......... .......... .......... .......... .......... 40% 28.5M 0s
#0 35.63   2350K .......... .......... .......... .......... .......... 41% 28.5M 0s
#0 35.63   2400K .......... .......... .......... .......... .......... 42% 21.5M 0s
#0 35.64   2450K .......... .......... .......... .......... .......... 43% 25.7M 0s
#0 35.64   2500K .......... .......... .......... .......... .......... 44% 28.5M 0s
#0 35.64   2550K .......... .......... .......... .......... .......... 45% 28.3M 0s
#0 35.64   2600K .......... .......... .......... .......... .......... 46% 27.3M 0s
#0 35.64   2650K .......... .......... .......... .......... .......... 47% 29.0M 0s
#0 35.64   2700K .......... .......... .......... .......... .......... 47% 28.2M 0s
#0 35.65   2750K .......... .......... .......... .......... .......... 48% 28.6M 0s
#0 35.65   2800K .......... .......... .......... .......... .......... 49% 19.9M 0s
#0 35.65   2850K .......... .......... .......... .......... .......... 50% 28.5M 0s
#0 35.65   2900K .......... .......... .......... .......... .......... 51% 28.2M 0s
#0 35.65   2950K .......... .......... .......... .......... .......... 52% 25.7M 0s
#0 35.66   3000K .......... .......... .......... .......... .......... 53% 31.5M 0s
#0 35.66   3050K .......... .......... .......... .......... .......... 53% 28.3M 0s
#0 35.66   3100K .......... .......... .......... .......... .......... 54% 25.6M 0s
#0 35.66   3150K .......... .......... .......... .......... .......... 55% 28.3M 0s
#0 35.66   3200K .......... .......... .......... .......... .......... 56% 21.6M 0s
#0 35.66   3250K .......... .......... .......... .......... .......... 57% 28.3M 0s
#0 35.67   3300K .......... .......... .......... .......... .......... 58% 28.5M 0s
#0 35.67   3350K .......... .......... .......... .......... .......... 59% 25.5M 0s
#0 35.67   3400K .......... .......... .......... .......... .......... 60% 28.1M 0s
#0 35.67   3450K .......... .......... .......... .......... .......... 60% 28.4M 0s
#0 35.67   3500K .......... .......... .......... .......... .......... 61% 28.5M 0s
#0 35.68   3550K .......... .......... .......... .......... .......... 62%  813K 0s
#0 35.74   3600K .......... .......... .......... .......... .......... 63% 34.5M 0s
#0 35.74   3650K .......... .......... .......... .......... .......... 64%  221M 0s
#0 35.74   3700K .......... .......... .......... .......... .......... 65%  230M 0s
#0 35.74   3750K .......... .......... .......... .......... .......... 66%  223M 0s
#0 35.74   3800K .......... .......... .......... .......... .......... 67% 20.8M 0s
#0 35.74   3850K .......... .......... .......... .......... .......... 67% 36.0M 0s
#0 35.74   3900K .......... .......... .......... .......... .......... 68%  222M 0s
#0 35.74   3950K .......... .......... .......... .......... .......... 69% 29.3M 0s
#0 35.75   4000K .......... .......... .......... .......... .......... 70% 62.8M 0s
#0 35.75   4050K .......... .......... .......... .......... .......... 71%  211M 0s
#0 35.75   4100K .......... .......... .......... .......... .......... 72% 72.1M 0s
#0 35.75   4150K .......... .......... .......... .......... .......... 73%  160M 0s
#0 35.75   4200K .......... .......... .......... .......... .......... 74%  157M 0s
#0 35.75   4250K .......... .......... .......... .......... .......... 74%  153M 0s
#0 35.75   4300K .......... .......... .......... .......... .......... 75%  204M 0s
#0 35.75   4350K .......... .......... .......... .......... .......... 76%  216M 0s
#0 35.75   4400K .......... .......... .......... .......... .......... 77%  165M 0s
#0 35.75   4450K .......... .......... .......... .......... .......... 78%  229M 0s
#0 35.75   4500K .......... .......... .......... .......... .......... 79%  220M 0s
#0 35.75   4550K .......... .......... .......... .......... .......... 80% 17.2M 0s
#0 35.75   4600K .......... .......... .......... .......... .......... 80%  197M 0s
#0 35.75   4650K .......... .......... .......... .......... .......... 81%  187M 0s
#0 35.75   4700K .......... .......... .......... .......... .......... 82%  223M 0s
#0 35.75   4750K .......... .......... .......... .......... .......... 83%  257M 0s
#0 35.75   4800K .......... .......... .......... .......... .......... 84%  192M 0s
#0 35.75   4850K .......... .......... .......... .......... .......... 85%  212M 0s
#0 35.75   4900K .......... .......... .......... .......... .......... 86%  190M 0s
#0 35.75   4950K .......... .......... .......... .......... .......... 87%  226M 0s
#0 35.75   5000K .......... .......... .......... .......... .......... 87%  208M 0s
#0 35.75   5050K .......... .......... .......... .......... .......... 88%  229M 0s
#0 35.75   5100K .......... .......... .......... .......... .......... 89%  222M 0s
#0 35.75   5150K .......... .......... .......... .......... .......... 90%  252M 0s
#0 35.75   5200K .......... .......... .......... .......... .......... 91%  209M 0s
#0 35.75   5250K .......... .......... .......... .......... .......... 92%  261M 0s
#0 35.75   5300K .......... .......... .......... .......... .......... 93%  242M 0s
#0 35.75   5350K .......... .......... .......... .......... .......... 94%  227M 0s
#0 35.75   5400K .......... .......... .......... .......... .......... 94%  195M 0s
#0 35.76   5450K .......... .......... .......... .......... .......... 95%  226M 0s
#0 35.76   5500K .......... .......... .......... .......... .......... 96%  235M 0s
#0 35.76   5550K .......... .......... .......... .......... .......... 97%  194M 0s
#0 35.76   5600K .......... .......... .......... .......... .......... 98%  194M 0s
#0 35.76   5650K .......... .......... .......... .......... .......... 99%  224M 0s
#0 35.76   5700K .......... .......... .......... .......... ..        100%  149M=0.2s
#0 35.76 
#0 35.76 2023-03-13 18:14:33 (23.3 MB/s) - 'kustomize_v5.0.0_linux_amd64.tar.gz' saved [5880512/5880512]
#0 35.76 
#0 35.76 
#0 35.76 gzip: stdin: unexpected end of file
#0 35.76 tar: Child returned status 1
#0 35.76 tar: Error is not recoverable: exiting now
#0 35.76 Makefile:25: recipe for target 'install-kustomize' failed
#0 35.76 make: *** [install-kustomize] Error 2
------
deploy.Dockerfile:8
--------------------
   7 |     
   8 | >>> RUN apt update \
   9 | >>>     && apt install --yes \
  10 | >>>         git \
  11 | >>>         curl \
  12 | >>>         unzip \
  13 | >>>         tar \
  14 | >>>         make \
  15 | >>>         sudo \
  16 | >>>         vim \
  17 | >>>         wget \
  18 | >>>     && git clone https://github.com/awslabs/kubeflow-manifests.git \
  19 | >>>     && cd kubeflow-manifests \
  20 | >>>     && git checkout ${AWS_RELEASE_VERSION} \
  21 | >>>     && git clone --branch ${KUBEFLOW_RELEASE_VERSION} https://github.com/kubeflow/manifests.git upstream \
  22 | >>>     && make install-tools
  23 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c apt update     && apt install --yes         git         curl         unzip         tar         make         sudo         vim         wget     && git clone https://github.com/awslabs/kubeflow-manifests.git     && cd kubeflow-manifests     && git checkout ${AWS_RELEASE_VERSION}     && git clone --branch ${KUBEFLOW_RELEASE_VERSION} https://github.com/kubeflow/manifests.git upstream     && make install-tools" did not complete successfully: exit code: 2
```

**Steps To Reproduce**
1. Create a dockerfile to reproduce:  
**deploy.Dockerfile**
```dockerfile
FROM ubuntu:18.04

ARG KUBEFLOW_RELEASE_VERSION
ARG AWS_RELEASE_VERSION

WORKDIR /tmp/

RUN apt update \
    && apt install --yes \
        git \
        curl \
        unzip \
        tar \
        make \
        sudo \
        vim \
        wget \
    && git clone https://github.com/awslabs/kubeflow-manifests.git \
    && cd kubeflow-manifests \
    && git checkout ${AWS_RELEASE_VERSION} \
    && git clone --branch ${KUBEFLOW_RELEASE_VERSION} https://github.com/kubeflow/manifests.git upstream \
    && make install-tools


WORKDIR /tmp/kubeflow-manifests/deployments/cognito-rds-s3/terraform

# Disable automatic subdomain creation since our root domain is not on AWS Route53
ARG CREATE_SUBDOMAIN="false"

ARG CLUSTER_NAME

ARG CLUSTER_REGION

ARG EKS_VERSION

# Name of an existing Route53 root domain (e.g. example.com)
ARG ROOT_DOMAIN
# Name of the subdomain to create (e.g. platform.example.com)
ARG SUBDOMAIN

ARG USER_POOL_NAME

ARG USE_RDS="true"

ARG USE_S3="true"

ARG USE_COGNITO="true"

ARG LOAD_BALANCER_SCHEME=internet-facing

ARG NOTEBOOK_ENABLE_CULLING=true

ARG NOTEBOOK_CULL_IDLE_TIMEOUT_SECONDS=120

ARG NOTEBOOK_IDLENESS_CHECK_PERIOD=10

ARG SECRET_RECOVERY_WINDOW_IN_DAYS=30

ARG NODE_INSTANCE_TYPE

ARG AWS_ACCESS_KEY_ID
ENV AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}

ARG AWS_SECRET_ACCESS_KEY
ENV AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}

ARG AWS_DEFAULT_REGION
ENV AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}

RUN    echo "minio_aws_access_key_id=\"${MINIO_AWS_ACCESS_KEY_ID}\"" >> sample.auto.tfvars \
    && echo "minio_aws_secret_access_key=\"${MINIO_AWS_SECRET_ACCESS_KEY}\"" >> sample.auto.tfvars \
    && echo "create_subdomain=\"${CREATE_SUBDOMAIN}\"" >> sample.auto.tfvars \
    && echo "cluster_name=\"${CLUSTER_NAME}\"" >> sample.auto.tfvars \
    && echo "cluster_region=\"${CLUSTER_REGION}\"" >> sample.auto.tfvars \
    && echo "eks_version=\"${EKS_VERSION}\"" >> sample.auto.tfvars \
    && echo "generate_db_password=\"true\"" >> sample.auto.tfvars \
    && echo "aws_route53_root_zone_name=\"${ROOT_DOMAIN}\"" >> sample.auto.tfvars \
    && echo "aws_route53_subdomain_zone_name=\"${SUBDOMAIN}\"" >> sample.auto.tfvars \
    && echo "cognito_user_pool_name=\"${USER_POOL_NAME}\"" >> sample.auto.tfvars \
    && echo "use_rds=\"${USE_RDS}\"" >> sample.auto.tfvars \
    && echo "use_s3=\"${USE_S3}\"" >> sample.auto.tfvars \
    && echo "use_cognito=\"${USE_COGNITO}\"" >> sample.auto.tfvars \
    && echo "load_balancer_scheme=\"${LOAD_BALANCER_SCHEME}\"" >> sample.auto.tfvars \
    && echo "notebook_enable_culling=\"${NOTEBOOK_ENABLE_CULLING}\"" >> sample.auto.tfvars \
    && echo "notebook_cull_idle_time=\"${NOTEBOOK_CULL_IDLE_TIMEOUT_SECONDS}\"" >> sample.auto.tfvars \
    && echo "notebook_idleness_check_period=\"${NOTEBOOK_IDLENESS_CHECK_PERIOD}\"" >> sample.auto.tfvars \
    && echo "secret_recovery_window_in_days=\"${SECRET_RECOVERY_WINDOW_IN_DAYS}\"" >> sample.auto.tfvars \
    && echo "node_instance_type=\"${NODE_INSTANCE_TYPE}\"" >> sample.auto.tfvars \
    && terraform init \
    && terraform plan

CMD ["make", "deploy"]
```
2. build the image:  
```shel
docker build \
    --build-arg AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
    --build-arg AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
    --build-arg AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION \
    --build-arg KUBEFLOW_RELEASE_VERSION=$KUBEFLOW_RELEASE_VERSION \
    --build-arg AWS_RELEASE_VERSION=$AWS_RELEASE_VERSION \
    --build-arg MINIO_AWS_ACCESS_KEY_ID=$MINIO_AWS_ACCESS_KEY_ID \
    --build-arg MINIO_AWS_SECRET_ACCESS_KEY=$MINIO_AWS_SECRET_ACCESS_KEY \
    --build-arg CLUSTER_NAME=$CLUSTER_NAME \
    --build-arg CLUSTER_REGION=$CLUSTER_REGION \
    --build-arg EKS_VERSION=$EKS_VERSION \
    --build-arg ROOT_DOMAIN=$ROOT_DOMAIN \
    --build-arg SUBDOMAIN=$SUBDOMAIN \
    --build-arg USER_POOL_NAME=$USER_POOL_NAME \
    --build-arg NODE_INSTANCE_TYPE=$NODE_INSTANCE_TYPE \
    -t kf-deployment \
    . \
    -f deploy.Dockerfile
```  
3. Notice the error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Environment**
* Kubernetes version
* Using EKS (yes/no), if so version? 1.25
* Kubeflow version : `KUBEFLOW_RELEASE_VERSION=v1.7.0-rc.2`
* AWS build number: `AWS_RELEASE_VERSION=9fcd434257fef6d5bfa1667647cd67fae53cb540`
* AWS service targeted (S3, RDS, etc.): Cognito-RDS-S3

--- a/Makefile
+++ b/Makefile
@@ -23,17 +23,17 @@ install-kubectl:
 
 install-kustomize:
 	$(eval KUSTOMIZE_VERSION:=5.0.0)
-	wget https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v$(KUSTOMIZE_VERSION)/kustomize_v$(KUSTOMIZE_VERSION)_linux_amd64.tar.gz | tar xz -C /tmp
+	curl --silent --location "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv$(KUSTOMIZE_VERSION)/kustomize_v$(KUSTOMIZE_VERSION)_linux_amd64.tar.gz" | tar xz -C /tmp
 	chmod +x /tmp/kustomize
-	sudo mv /tmp/kustomize usr/local/bin/kustomize
+	sudo mv /tmp/kustomize /usr/local/bin/kustomize
 	kustomize version
 
 install-yq:
 	$(eval YQ_VERSION:=v4.26.1)
-	wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64.tar.gz -O - | tar xz 
-	sudo mv yq_linux_amd64 /usr/bin/yq
-	rm install-man-page.sh
-	rm yq.1
+	curl --silent --location "https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64.tar.gz" | tar xz -C /tmp
+	sudo mv /tmp/yq_linux_amd64 /usr/bin/yq
+	rm /tmp/install-man-page.sh
+	rm /tmp/yq.1
 	yq --version
 
 install-jq:

